connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -559,6 +576,8 @@ private void resetDelay() {
         log.debug("Previous rounded down (floor) average number of tasks per worker {}", totalActiveTasksNum / existingWorkersNum);
         int floorTasks = totalActiveTasksNum / totalWorkersNum;
         log.debug("New rounded down (floor) average number of tasks per worker {}", floorTasks);
+        int ceilTasks = (int) Math.ceil((float) totalActiveTasksNum / totalWorkersNum);
+        log.debug("New rounded down (ceil) average number of tasks per worker {}", ceilTasks);
```

Comment:
What do you think about combining these log messages?
```suggestion
        int ceilTasks = (int) Math.ceil((float) totalActiveTasksNum / totalWorkersNum);
        log.debug("New average number of tasks per worker: floor={}, ceiling={}", floorTasks, ceilTasks);
```